firebase/firestore-py/lib/services/auth.py

```diff
@@ -9,12 +9,6 @@ def get_user(email):
 	try: return auth.get_user_by_email(email)
 	except NotFoundError: return create_user(email)
 
-def list_users(): 
```

Comment:
?

Reply:
I will check later, but I think the reason I did this was because I moved it elsewhere actually

Reply:
I don't think I moved it anywhere actually and infact there is another file that calls this function. Must've been a mistake. 

Good catch.

Reply:
Ah okay, feel free to merge after